src/website/views/components/modal.pug

```diff
@@ -43,6 +43,78 @@ h4 Base
       </div>
     </div>
 
+h4 Scrollable
+p.-text
+  | If the content height is bigger than the content modal container
+  | height, we can make the content scrollable.
+h5 Requirements
+p.-text
+  | Content Modal must have the<code>max-height</code> defined.
```

Comment:
Change to:

`A <code>max-height</code> must be defined on <code>a-modal__content</code>. Use max-height utility classes such as <code>-mh--400</code> or <code>-mh--480</code>, or define your own value.`

Reply:
done